### DISCUSSION TITLE
特殊字符☑识别不了

### DISCUSSION THREAD

**Discussion by blvyoucan** [PaddlePaddle/PaddleOCR#13737] [Q&A]
### 🐛 Bug (问题描述)

字符☑识别不了（不在字典中的字符），还有□会识别成口

**Comment by GreatV** [comment]
参考 https://paddlepaddle.github.io/PaddleOCR/ppocr/model_train/training.html#32

**Reply by blvyoucan** [reply]
识别要上百万的训练数据，这个有点困难。。。

**Reply by Miller-em** [reply]
> 参考 https://paddlepaddle.github.io/PaddleOCR/ppocr/model_train/training.html#32

这个链接已经打不开了，能重新给一个吗
